app/assets/stylesheets/searchworks4/search-results.css

```diff
@@ -164,3 +164,25 @@ details[open] {
     -webkit-text-stroke: 0.45px;
   }
 }
+
+.collection-info {
+  background-color: var(--stanford-fog-light);
+
+  .finding-aid a {
+    --bs-link-decoration: none;
+
+    &::after {
+      content: '\F144';
```

Comment:
Can we try this with #5595, so that we can use the typical ways of rendering icons?